requirements.txt

```diff
@@ -8,3 +8,4 @@ sphinx-rtd-theme==0.4.3
 google-api-python-client==1.10.1
 google-auth-httplib2==0.0.4
 google-auth-oauthlib==0.4.2
+sqlalchemy==1.4.28
```

Comment:
sqlalchemy needs to be included in the setup.py and unpinned.

Reply:
done